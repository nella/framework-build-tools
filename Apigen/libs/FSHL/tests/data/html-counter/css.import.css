@import url("http://www.site.com/style.css");