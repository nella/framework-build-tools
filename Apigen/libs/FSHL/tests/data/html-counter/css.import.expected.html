<html><head><meta http-equiv="content-type" content="text/html; charset=utf-8" /><link rel="stylesheet" type="text/css" href="../../../style.css" media="all" /></head><body><pre>
<span class="line">1: </span><span class="css-at-rule">@import</span> <span class="css-func">url(</span><span class="css-value">&quot;http://www.site.com/style.css&quot;</span><span class="css-func">)</span>;
</pre></body></html>